recipes/servers.md

```diff
@@ -86,7 +86,7 @@ http.createServer(app).listen(3000);
 ## lite-server
 
 https://github.com/johnpapa/lite-server
-[![GitHub stars](https://img.shields.io/github/stars/johnpapa/lite-server.svg?style=social&label=Star)](https://github.com/johnpapa/lite-server) ([example source](https://github.com/johnpapa/lite-server/issues/61#issuecomment-205997607))
+[![GitHub stars](https://img.shields.io/github/stars/johnpapa/lite-server.svg?style=social&label=Star)](https://github.com/johnpapa/lite-server)
```

Comment:
I removed the link because it points to an example that overwrites the default configs, which would not make sense anymore with the proposed modifications.